app/code/Magento/CatalogInventory/Api/Data/WarehouseRecordInterface.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Magento\CatalogInventory\Api\Data;
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Warehouse record interface
+ *
+ * It is used only to manage stock status
+ * qty of non virtual warehouse inventory
+ *
+ * @api
+ */
+interface WarehouseRecordInterface extends ExtensibleDataInterface
+{
+    /**
+     * Returns identifier of inventory record
+     *
+     * @return int
+     */
+    public function getId();
+
+    /**
+     * Returns identifier of assigned product
+     *
+     * @return int
+     */
+    public function getProductId();
+
+    /**
+     * Returns identifier of assigned inventory
+     *
+     * @return int
+     */
+    public function getInventoryId();
```

Comment:
Please, clarify: is it really supposed to be Inventory Id here, or Warehouse Id?